### PR TITLE
Added known issue for Pipelines 1.6.1 RN

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.9`, `enterprise-4.10`
- **JIRA issues**: [RHDEVDOCS-3571 RN for Pipelines 1.6.1](https://issues.redhat.com/browse/RHDEVDOCS-3571)
- **Preview pages**: [Release notes for Red Hat OpenShift Pipelines General Availability 1.6.1 > Known Issues](https://deploy-preview-40009--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes#known-issues-1-6-1_op-release-notes)
- **SME review**: @concaf    
- **Peer-review**: @Preeticp

### CODE REVIEW

**Review by concaf — APPROVED** [review]

**Comment by sm43** [comment]
LGTM

File: modules/op-release-notes-1-6.adoc [hunk 1]
```diff
@@ -251,7 +251,18 @@ time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
 [id="release-notes-1-6-1_{context}"]
 == Release notes for {pipelines-title} General Availability 1.6.1
 
-The following improvements are made in {pipelines-title} 1.6.1:
+[id="known-issues-1-6-1_{context}"]
+=== Known issues
+
+* After upgrading to {pipelines-title} 1.6.1 from an older version, Pipelines might enter an inconsistent state where you are unable to perform any operations (create/delete/apply) on Tekton resources (tasks and pipelines). For example, while deleting a resource, you might encounter the following error:
```
Comment (Preeticp) [inline modules/op-release-notes-1-6.adoc:257]: Pipelines might enter:
Small 'p' in pipelines
might or will?

**Pull request merged**